DESCRIPTION

```diff
@@ -1,19 +1,23 @@
 Package: consensusOV
 Type: Package
-Title: Gene expression-based subtype classifiers for high-grade serous ovarian cancer
-Version: 0.99.0
+Title: Gene expression-based subtype classification for high-grade serous ovarian cancer
+Version: 0.99.1
 Date: 2017-05-10
-Author: Gregory M Chen, Lavanya Kannan, Levi Waldron, Benjamin Haibe-Kains
+Author: Gregory M Chen, Lavanya Kannan, Ludwig Geistlinger, Benjamin Haibe-Kains, Levi Waldron
```

Comment:
Ordering of authors is incorrect, Benjamin should be listed last as corresponding author. This is the correct ordering:

Author: Gregory M Chen, Lavanya Kannan, Ludwig Geistlinger, Victor Kofia, Levi Waldron, Benjamin Haibe-Kains